### PR TITLE
Clean up

### PR DESCRIPTION
**Pull request by steven-solar**
This PR cleans up code, changes names to be more obvious and breaks js into helper functions (with comments!).

### CODE REVIEW

File: portfolio/src/main/webapp/script.js [hunk 1]
```diff
@@ -1,17 +1,8 @@
...
-//
-// Unless required by applicable law in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
 
+/** 
+ * Validates the user's inputted number of comments to be displayed, called on input.
+ * @return {boolean} Whether user input was a valid, postive integer.
```
Comment (VivekShankar) [inline portfolio/src/main/webapp/script.js:4]: nit: positive

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -161,10 +156,43 @@ function renderForm() {
   }
 }
 
+/**
+ * Creates and returns the auth message if user is logged in.
+ * @return {Document Element} Mesasge for logged in users.
```
Comment (VivekShankar) [inline portfolio/src/main/webapp/script.js:161]: nit: Message

File: portfolio/src/main/webapp/script.js [hunk 3]
```diff
@@ -161,10 +156,43 @@ function renderForm() {
...
+  authLink.href = loginUrl;
+  authMessage.appendChild(document.createTextNode("Logout "));
+  authMessage.appendChild(authLink);
+  authMessage.appendChild(document.createTextNode("."));
+  return authMessage;
+}
+
+/**
+ * Creates and returns the auth message if user is logged out.
+ * @return {Document Element} Mesasge for logged out users.
```
Comment (VivekShankar) [inline portfolio/src/main/webapp/script.js:176]: nit: Message

File: portfolio/src/main/webapp/script.js [hunk 4]
```diff
@@ -194,7 +222,12 @@ function makeMap() {
 let markers = [];
 let marker;
 
-function addPin(comment, time, i) {
+/** 
+ * Creates and drops a pin and info window for a given comment
+ * @param {comment} The comment to be displayed.
+ * @param {comment} The time delay for animating the pin drop.
```
Comment (VivekShankar) [inline portfolio/src/main/webapp/script.js:228]: Should this be number?

File: portfolio/src/main/webapp/script.js (hunk 1 above)
Comment (steven-solar) [reply portfolio/src/main/webapp/script.js:4]: fixed! Nice catch, someone should make a spell-check plugin for IDEs.

File: portfolio/src/main/webapp/script.js (hunk 4 above)
Comment (steven-solar) [reply portfolio/src/main/webapp/script.js:228]: It should! Thanks for catching that, changing the second param to "time".

**Pull request merged**